test/gtest/uct/cuda/test_cuda_ipc_md.cc

```diff
@@ -67,6 +122,34 @@ UCS_MT_TEST_P(test_cuda_ipc_md, multiple_mds, 8)
         rkey = unpack(md, i + 1);
         EXPECT_EQ(i, rkey.dev_num);
     }
+
+#if HAVE_CUDA_FABRIC
+    CUdeviceptr ptr;
+    CUmemoryPool mpool, q_mpool;
+    CUstream cu_stream;
+    CUmemFabricHandle fabric_handle;
+    CUresult cu_err;
+
+    alloc_mempool(&ptr, &mpool, &cu_stream, 64);
+    EXPECT_EQ(CUDA_SUCCESS, (cuPointerGetAttribute((void*)&q_mpool,
+                    CU_POINTER_ATTRIBUTE_MEMPOOL_HANDLE, ptr)));
+
+    cu_err = cuMemPoolExportToShareableHandle((void*)&fabric_handle, q_mpool,
+                                              CU_MEM_HANDLE_TYPE_FABRIC, 0);
+    free_mempool(&ptr, &mpool, &cu_stream);
+
+    if (cu_err == CUDA_SUCCESS) {
+        for (int64_t i = 0; i < 64; ++i) {
```

Comment:
why is `int_64_t` needed?

Reply:
well, this is to match `unpack_masync` function that takes `int64_t` as input arg